wagn/lib/wagn/tasks/wagn.rake

```diff
@@ -73,8 +74,8 @@ namespace :wagn do
 
   desc "migrate structure and cards"
   task :migrate =>:environment do
-    ENV['SCHEMA'] ||= "#{Wagn.root}/db/schema.rb"
-    
+    ENV['SCHEMA'] ||= "#{Cardio.gem_root}/db/schema.rb"
```

Comment:
For 1., then for the developer case, update in the gem and the dev will checking/checkout as needed.

And I meant 'write to the gem space' when I said 'gem file', not the Gemfile in the deck.

I'm not sure I get the terms quite yet: *rails decks* makes sense as a term, but it does involve decko-rails.  I might use *free decks* for the non-rails case (just model, maybe Sinatra, other web framework, or some other framework), which nobody is really considering yet.  We would know that case by the existence of a new glue gem or library (say decko-sinatra).

Maybe *deck app* for the current stand-alone rails app case.